// This file describes the protobuf for attacking services

syntax = "proto3";

package attack_service;

import "message_definitions.proto";

// Provides API for attacks on digital signatures
service DigitalSignatureAttack {
  /**
   * Provides function for attack on ECDSA nonce reuse vulnerability
   */
  rpc ecdsaReusedNonceAttack(messages.ReusedNonceAttackRequest)
                                      returns (messages.ReusedNonceAttackResponse) {};
}
